2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -45,15 +45,27 @@
 
   // feature flags booleans
   private boolean camera1Enable = false;
+  private boolean camera1Enabled = false;
```

Comment:
Small suggestion (a nit really) - these names are pretty similar and could be confusing. Would it be worth renaming to `camera1Initialized` etc?

Reply:
done